config/notifikasjon/dev-gcp.yml

```diff
@@ -2,6 +2,10 @@ kafkaPool: nav-dev
 azure:
   enabled: true
 ingress: https://helsearbeidsgiver-im-notifikasjon.intern.dev.nav.no
+#database: sak_oppgave
+#envFrom:
+#  - type: secret
+#    name: google-sql-im-notifikasjon
```

Comment:
Kommenterer dette inn når vi er enige om at databasen er klar til å opprettes.

Reply:
er det ikke mer logisk at databasen heter notifikasjon? Matcher bedre appen som eier den, og det kan jo på sikt legges andre ting der også

Reply:
Jeg tenkte at notifikasjon ikke er helt dekkende, siden notifikasjon egentlig bare er en "sideeffekt" av opprettelse av oppgave. Sånn sett er appen ikke så godt navngitt heller. Men for å være konsekvent så kan jeg endre det db-navnet.